_posts/en/newsletters/2018-12-18-newsletter.md

```diff
@@ -0,0 +1,200 @@
+  definite proposal with code can only truly become part of the Bitcoin
+  protocol if it has the support of a large number of economically
+  relevant full node users---and building such support could take years,
+  if it's achieved at all.
+
+- **LN protocol IRC meeting:** protocol developers for LN have agreed to
+  try converting their periodic meeting for developing the LN
+  specification from a Google Hangout to an IRC meeting after receiving
+  requests from several developers.  The [next meeting][ln irc meeting]
+  will be Tuesday, January 8th, at 19:00 (UTC).
+
+## Notable code changes
+
+*Notable code changes this week in [Bitcoin Core][core commits],
+[LND][lnd commits], [C-lightning][cl commits], and [libsecp256k1][secp
+commits].*
+
+- [Bitcoin Core #14573][] moves various miscellaneous options that
+  opened separate dialogues in the Bitcoin-Qt GUI to a new top-level
+  menu item labeled *Window*, hopefully making those options easier to
+  find and use.
+
+- [LND #1984][] adds a new `listunspent` RPC that lists each of the
+  wallet's unspent outputs.  It can take two parameters: (1) the minimum
+  number of confirmations the unspent output must have or (2) the maximum
+  it can have.  The minimum can be set to `0` to print unconfirmed
+  outputs.
+
+- [LND #2039][] adds the ability to get the status of the autopilot
+  functionality as well as allowing enabling or disabling it while the
+  program is running.  Autopilot is the ability of the software to
+  automatically suggest new channels to open when a user is first
+  connecting to LN or wants additional spending capacity.
+
+- [LND #2324][] updates and expands LND's installation documentation.
+
+- [C-Lightning #2155][] disables by default the
+  `option_data_loss_protect` feature described in the notable commits of
+  [newsletter #10][].  The feature wasn't working reliably, so it will
+  only be enabled for users that opt-in to experimental features.
+
+- [C-Lightning #2154][] now allows plugins to send log notifications
+  that will be written to the lightningd's log files.
+
+- [C-Lightning #2132][] adds significant inline code documentation for
+  `openingd`, the sub-server that handles opening new payment channels.
+
+- [C-Lightning #2161][] adds a small Python library and framework that
+  can be used for writing plugins.  It provides [function decorators][]
+  similar to those used by the popular [flask][] library that can be
+  used to tag functions as providing particular plugin interfaces, and
+  this information is automatically used to generate a plugin manifest.
+  The sample `helloworld.py` plugin has been updated to use this
+  library, reducing its size by 75% (111 lines to 28).
+
+## Holiday publication schedule
+
+Due to the holidays, the Optech Newsletter will adopting a modified
+publication schedule:
+
+- Friday, December 21st: a special 2018 year-in-review edition
+- Thursday, January 3rd: a regular newsletter providing updates through
```

Comment:
I'm happy to review and publish on this schedule if you want. I just worry that the Jan 8th issue is going to be a little thin given it'll only be covering two weekdays (Jan 3rd and 4th). It's likely to be a slow news period anyway, so I suggest we just cancel the Thurs, Jan 3rd issue and skip straight to Tues Jan 8th. Thoughts?

Reply:
An extra week off sounds good to me!

Reply:
In that case, we may want to push the year-in-review back to ~28th Dec to better bisect the regular issues. Very happy to publish whenever you want though.

Reply:
Good idea!  Updated.